src/dials/command_line/dials_import.py

```diff
@@ -115,6 +115,10 @@ def _pickle_load(fh):
       .help = "A directory with images"
       .multiple = True
 
+    split = None
+      .type = ints(size=3)
+      .help = "Scan split instructions"
```

Comment:
Let's make this a bit more concrete: "Scan split instructions of three numbers: start, end, frames_per_chunk. start and end are 1-indexed."